web/src/components/apps/AppLicense.tsx

```diff
@@ -472,6 +472,14 @@ const AppLicenseComponent = () => {
                       enabled{" "}
                     </span>
                   ) : null}
+                  {isEmbeddedCluster &&
+                  !appLicense?.isEmbeddedClusterMultinodeDisabled ? (
```

Comment:
we can leave out the ternary and use logical `&&` operator here for improved readability